.circleci/config.yml

```
@@ -65,6 +65,33 @@ jobs:
             - store_test_results:
                 path: test-reports
 
+    macos_compat_build:
+        macos:
+            xcode: 12.0
+        environment:
+            - PYTORCH_REF=master
```

This should probably match the `pytorch_ref` we test for all of the other compat-tests

```suggestion
            - PYTORCH_REF=v1.7.0-rc2
```